benchmarks/src/main/scala/zio/redis/lists/BlPopBenchmarks.scala

```diff
@@ -0,0 +1,65 @@
+package zio.redis.lists
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+import zio.ZIO
+import zio.duration._
+import zio.redis.{ BenchmarkRuntime, blPop, rPush }
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Measurement(iterations = 15)
+@Warmup(iterations = 15)
+@Fork(2)
+class BlPopBenchmarks extends BenchmarkRuntime {
+  @Param(Array("500"))
+  var count: Int = _
+
+  private var items: List[String] = _
+
+  private val key = "test-list"
+
+  @Setup(Level.Invocation)
```

Comment:
You don't get anything with invocation level here. The same applies for setup calls in other benchmarks.

Reply:
It's to populate the list each invocation, after it has been emptied by popping/deletion commands. The invocation level is used in teardown calls as well, to delete the elements of the list so that the push/insert calls don't become too expensive, and affect every subsequent benchmark.

Reply:
Good catch. As discussed in chat, please make sure to adjust the existing benchmarks in the same way in the follow-up.